pkg/config/connections_test.go

```diff
@@ -16,7 +16,7 @@ var _ = Describe("Connections conf", func() {
 
 	BeforeEach(func() {
 		dir := GinkgoT().TempDir()
-		connectionsConfFile = filepath.Join(dir, "connections.conf")
+		connectionsConfFile = filepath.Join(dir, "connections.json")
```

Comment:
maybe I missed something, but should this be "podman-connections.json"  ?

Reply:
yes this name does not matter at all, it could be anything